spec/smoke_tests/weird_rspec_spec.rb

```diff
@@ -11,6 +11,9 @@
   let (:foo) { 1 }
   let!   (:bar){}
 
+  bar = -> {}
+  let(:foo, &bar)
```

Comment:
I love that this file exists. A really clever way to ensure that future rule changes all account for weird ruby syntax.